src/index.js

```diff
@@ -184,6 +186,11 @@ addNewListForm.addEventListener('submit', (e) => {
 //   saveAndRender();
 // });
```

Comment:
- [ ] For best practices and clean code reasons, kindly ensure that you remove all the commented and unused code lines from your code base.

-----------------------------------------------------------------------------------------------------------------------------------------------------
### Notice
Hi @Pi1998 
Just notice that, I am considering this as 1st review as I can see that you are repeating the module.
Good luck and happy coding